standard/InstallHeaders.cmake

```diff
@@ -40,31 +40,20 @@ function(install_headers)
       get_filename_component(src_bin ${src} ABSOLUTE BASE_DIR ${CMAKE_CURRENT_BINARY_DIR})
       if(EXISTS ${src_src})
         set(src ${src_src})
+        file(RELATIVE_PATH src_rel ${CMAKE_CURRENT_SOURCE_DIR} ${src})
+        get_filename_component(src_rel ${src_rel} DIRECTORY)
+        set(actual_rel ${src_rel})
       elseif(EXISTS ${src_bin})
         set(src ${src_bin})
+        file(RELATIVE_PATH bin_rel ${CMAKE_CURRENT_BINARY_DIR} ${src})
```

Comment:
Since one checks the relation to `${CMAKE_CURRENT_SOURCE_DIR}` while the other `${CMAKE_CURRENT_BINARY_DIR}`, such a three-line function would require at least two arguments.

We probably wouldn't collapse the `EXISTS` check, since it's in the form of the if-elseif-else logic.

Can make this change, but want to double-check that I understand your suggestion.